src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/graphs/KBestHaplotype.java

```diff
@@ -9,19 +10,19 @@
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
  */
-public final class KBestHaplotype extends Path<SeqVertex,BaseEdge>{
+public final class KBestHaplotype<T extends BaseVertex, E extends BaseEdge> extends Path<T, E>{
```

Comment:
Why not use `V` instead of `T`?  And actually, there's no law against parameterizing the class with more than a single character i.e. `<VERTEX extends BaseVertex, EDGE extends BaseEdge>`.  I don't insist on that, though.

Reply:
You are right about V. Looking at google styleguide (which we only really loosely follow but its a good example of java style) they would want the type to be specified in the format `EdgeT`, `VertexT`, etc... I'm going to stick with the single letters for now and ask if there are other opinions on the matter.